src/borg/remote.py

```diff
@@ -306,6 +316,26 @@ def ssh_cmd(self, location):
             args.append('%s' % location.host)
         return args
 
+    def write_bandwith_limited(self, to_send):
+        if self.ratelimit:
+            current = time.monotonic()
```

Comment:
via release notes and linked bugtracker:
python 3.3 Availability: Windows, Mac OS X, Linux, FreeBSD, OpenBSD, Solaris.

Do we care about something else? i.e. does this need better error handling?